src/util/regulation.rs

```diff
@@ -88,7 +88,7 @@ impl Regulation {
 
         for file in &res.files {
             let path = PathBuf::from_str(&file.name).unwrap();
-            let file_name = path.file_stem().unwrap().to_str().unwrap();
+            let file_name = path.file_stem().unwrap().to_str().unwrap().split("\\").last().expect("Could not find content at end of path");
```

Comment:
```suggestion
            let file_name = &file.name.split("\\").last().expect("Could not locate file name").split(".").nth(0).expect("Could not locate file name without extension");
```

Reply:
@ClayAmore What do you think of these?

Reply:
It works in my VM

Reply:
Gus voice: It's acceptable. 

Have to circle back to this and figure out why file_stem couldn't strip the path on linux.